tests/unit/plugins/modules/test_info.py

```diff
@@ -640,6 +644,3 @@ def test_get_facts_event_groups_api_exception(self, gatherfacts_module_mock, gat
             gatherfacts_module_mock.perform_module_operation()
         assert MockGatherfactsApi.get_gather_facts_error_response(
             gather_subset) in gatherfacts_module_mock.module.fail_json.call_args[1]['msg']
-
```

Comment:
Why this has been removed?

Reply:
Just for testing few things, i had removed that piece of code. Added it back.